roles/scalelite/templates/env.j2

```diff
@@ -2,8 +2,14 @@
 SECRET_KEY_BASE={{ scalelite_secret_key_base }}
 LOADBALANCER_SECRET={{ scalelite_api_key }}
 
-### Required by Scalelite and Greenlight
+### Required by Scalelite, Greenlight and Postgres Exporter
 POSTGRES_PASSWORD={{ scalelite_postgres_password }}
+POSTGRES_SCALELITE_PASSWORD={{ scalelite_postgres_scalelite_password }}
+POSTGRES_GREENLIGHT_V3_PASSWORD={{ scalelite_postgres_greenlight_password }}
+POSTGRES_EXPORTER_PASSWORD={{ scalelite_postgres_exporter_password }}
+{% for instance in scalelite_greenlight_extra_instances %}
+POSTGRES_GREENLIGHT_{{ instance.name|upper }}_V3_PASSWORD={{ instance.postgres_password }}
```

Comment:
We should use loop_control to prevent the password from being printed when iterating over the Greenlight instances list

Reply:
We currently iterate over the list in two tasks. One already defined a label. I added it to the other one as well. The loops in the templates itself shouldn't be a problem, right (they don't seem to generate any output)?